src/components/Button/Button.js

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from "react";
+
+import { StyledButton, theme } from "./Button.styled";
+import { ThemeProvider } from "styled-components";
+
+const Button = ({children, variant, size, disabled, active}) => {
+    const [isActive, setIsActive] = useState(false);
+
+    return (
+        <ThemeProvider theme={theme}>
```

Comment:
Motyw najlepiej zdefiniować jak najwyżej w strukturze komponentów (np. w App), aby nie trzeba było go definiować wielokrotnie dla każdego elementu. Rozumiem, że w tym zadaniu było tak wygodniej, ale już w projekcie nalezy to zrobić inaczej ;)